llvm/test/ThinLTO/X86/dtlto/imports.test

```diff
@@ -0,0 +1,77 @@
+# Check that DTLTO creates imports lists correctly.
+
+RUN: rm -rf %t && split-file %s %t && cd %t
+
+# Compile bitcode.
+RUN: opt -thinlto-bc 0.ll -o 0.bc -O2
+RUN: opt -thinlto-bc 1.ll -o 1.bc -O2
+
+# Define a substitution to share the common DTLTO arguments. Note that the use
+# of validate.py will cause a failure as it does not create output files.
+DEFINE: %{command} = llvm-lto2 run 0.bc 1.bc -o t.o \
+DEFINE:    -dtlto \
+DEFINE:    -dtlto-distributor=%python \
+DEFINE:    -thinlto-distributor-arg=%llvm_src_root/utils/dtlto/validate.py \
+DEFINE:    -thinlto-distributor-arg=0.bc \
+DEFINE:    -thinlto-distributor-arg=1.bc \
+DEFINE:    -thinlto-emit-indexes \
+DEFINE:    -r=0.bc,g,px \
+DEFINE:    -r=1.bc,f,px \
+DEFINE:    -r=1.bc,g
+
+# We expect an import from 0.o into 1.o but no imports into 0.o. Check that the
+# expected input files have been added to the JSON.
+RUN: not %{command} >out.log 2>&1
+RUN: FileCheck --input-file=out.log %s --check-prefixes=INPUTS,ERR
+
+INPUTS:      "primary_input": [
+INPUTS-NEXT:   "0.bc"
+INPUTS-NEXT: ]
+INPUTS:      "imports": []
+INPUTS:      "primary_input": [
+INPUTS-NEXT:   "1.bc"
+INPUTS-NEXT: ]
+INPUTS:      "imports": [
+INPUTS-NEXT:   "0.bc"
+INPUTS-NEXT: ]
+
+# This check ensures that we have failed for the expected reason.
+ERR: failed: DTLTO backend compilation: cannot open native object file:
+
+# Check that imports files have not been created.
+RUN: ls | FileCheck %s --check-prefix=NOINDEXFILES
+NOINDEXFILES-NOT: imports
+
+# Check that imports files are not created with -save-temps.
+RUN: not %{command} -save-temps 2>&1 \ 
+RUN:   | FileCheck %s --check-prefixes=ERR
+RUN: ls | FileCheck %s --check-prefix=NOINDEXFILES
+NOINDEXFILES-NOT: imports
+
+# Check that imports files are created with -thinlto-emit-imports.
+RUN: not %{command} -thinlto-emit-imports 2>&1 \ 
+RUN:   | FileCheck %s --check-prefixes=ERR
+RUN: ls | FileCheck %s --check-prefix=INDEXFILES
```

Comment:
Similar to above suggest s/INDEX/IMPORT/

Reply:
Thanks. Fixed.